reporting/config/services/nifi/scripts/preload.sh

```diff
@@ -8,6 +8,14 @@ export PROC_GROUPS_DIR="$WORKING_DIR/preload/process-groups"
 export IMPORTED_PROC_GROUPS_DIR="/tmp/nifi-preload/process-groups"
 export NIFI_UP_RETRY_COUNT=240
 
+: ${POSTGRES_PASSWORD:?"Need to set POSTGRES_PASSWORD"}
+: ${AUTH_SERVER_CLIENT_ID:?"Need to set AUTH_SERVER_CLIENT_ID"}
+: ${AUTH_SERVER_CLIENT_SECRET:?"Need to set AUTH_SERVER_CLIENT_SECRET"}
+: ${TRUSTED_HOSTNAME:?"Need to set TRUSTED_HOSTNAME"}
+: ${OL_ADMIN_USERNAME:?"Need to set OL_ADMIN_USERNAME"}
+: ${OL_ADMIN_PASSWORD:?"Need to set OL_ADMIN_PASSWORD"}
+: ${OL_BASE_URL:?"Need to set OL_BASE_URL"}
```

Comment:
I would say if you want to use **measure** and **measure report** flows then you should provide those FHIR credentials. Otherwise, it depends if you need those values in your flows.

Reply:
Ok, so let leave it as it is